di_website/settings/base.py

```diff
@@ -32,6 +32,8 @@
     'di_website.common',
     'di_website.vacancies',
     'di_website.blog',
+    'di_website.events',
+    'di_website.whatwedo',
```

Comment:
How this magically returned, I have no idea... Will remove it again before approval.